reg_tests/ice_blend/driver.jet.sh

```diff
@@ -46,7 +46,7 @@ export COPYGB=/lfs4/HFIP/emcda/George.Gayno/ufs_utils.git/jet_port/grib_util/cop
 export COPYGB2=/lfs4/HFIP/emcda/George.Gayno/ufs_utils.git/jet_port/grib_util/copygb2
 export CNVGRIB=/apps/cnvgrib/1.4.0/bin/cnvgrib
```

Comment:
@GeorgeGayno-NOAA, is it possible to change the COPYGB and COPYGB2 locations to somewhere more general on Jet?  Do you know if these utilities exist in a base directory somewhere?  Or can we build them in our role.ufsutils directory?  I figured I'd check, since we already moved the baselines out of your personal directory.

Reply:
Good catch. Those utilities are part of the hpc-stack (module load grib_util/1.2.2). I will update the scripts to use them.